lib/Cake/Model/Datasource/CakeSession.php

```diff
@@ -552,6 +552,12 @@ protected static function _configureSession() {
 
 		if (!empty($sessionConfig['handler'])) {
 			$sessionConfig['ini']['session.save_handler'] = 'user';
+
+			// In PHP7.2.0+ session.save_handler can't be set to 'user' by the user.
+			// https://github.com/php/php-src/commit/a93a51c3bf4ea1638ce0adc4a899cb93531b9f0d
+			if (version_compare(PHP_VERSION, '7.2.0', '>=')) {
+				unset($sessionConfig['ini']['session.save_handler']);
+			}
```

Comment:
I referred to the following code. However, I think it is PHP 7.2 or higher that this setting cannot be done

https://github.com/cakephp/cakephp/blob/8b3905ad6f1d2f17d20d495e12e9947bd69644fb/src/Network/Session.php#L111-L114

- session.c on PHP 7.1 branch: https://github.com/php/php-src/blob/PHP-7.1/ext/session/session.c#L552-L577
- session.c on PHP 7.2 branch: https://github.com/php/php-src/blob/PHP-7.2/ext/session/session.c#L559-L563

Reply:
I think you have the correct code here. There is an open issue for database sessions in PHP7.2 (#11628)